### DISCUSSION TITLE
Plans for OPDS-PSE server side sync

### DISCUSSION THREAD

**Discussion by m-amazirh** [gotson/komga#1771] [Ideas]
Hi,

an OPDS-PSE server side sync feature was implemented in https://github.com/gotson/komga/issues/710 but was reverted by this commit e55f4936 .

Are there any plans to bring the feature back ?

Thanks

**Comment by gotson** [comment]
No, because it doesn't work. Clients often cache a few pages in advance, screwing the read progress. The way forward is OPDS 2, which has native read progression.
